live/eks/terragrunt.hcl

```diff
@@ -0,0 +1,25 @@
+terraform {
+  source = "git::https://github.com/softwaremill/terraform-eks-bootstrap//"
```

Comment:
This is a "latest" version of this module.
IMO it's a good idea to create a release (version) and pin this particular version here.